egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -11,6 +11,7 @@ mkdir -p $out_dir
 echo "Download WHAM noises into $out_dir"
 # If downloading stalls for more than 20s, relaunch from previous state.
 wget -c --tries=0 --read-timeout=20 https://storage.googleapis.com/whisper-public/wham_noise.zip -P $out_dir
+unzip $out_dir/wham_noise.zip -d $out_dir
 
 echo "Download WHAM scripts into $out_dir"
 wget https://storage.googleapis.com/whisper-public/wham_scripts.tar.gz -P $out_dir
```

Comment:
Moving this 
```
mkdir -p $out_dir/wham_scripts
tar -xzvf $out_dir/wham_scripts.tar.gz -C $out_dir/wham_scripts
mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
```
To this
```
tar -xzvf $out_dir/wham_scripts.tar.gz -C $out_dir
mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
```
sounds ok to you?

Reply:
cool